pyproject.toml

```diff
@@ -74,7 +74,7 @@ test = [
     "vcrpy<5",
 ]
 urllib3 = ["urllib3>=1.26"]
-validation = ["jsonschema>=4.0.1"]
+validation = ["jsonschema>=4.0.1,<4.18"]
```

Comment:
```suggestion
# jsonschema v4.18.2 breaks validation, and it feels safer to set a ceiling rather than just skip this version. The ceiling should be removed when the v4.18 lineage has settled down and feels safer.
validation = ["jsonschema>=4.0.1,<4.18"]
```